### PR TITLE
Send email for password reset

### PR DESCRIPTION
**Pull request by meganniu**
### Changes
+ Moved html for `PasswordResetEmailSentModal` to `SendResetPasswordEmailModal`
  + Show the html for `PasswordResetEmailSentModal` if password reset email is sent in `SendResetPasswordEmailModal.tsx`
  + Deleted `PasswordResetEmailSentModal.tsx`
+ Added auth function to send password reset modal

### Screenshots
Screenshot from 2021-04-11 22-55-00
Screenshot from 2021-04-11 22-56-15
Screenshot from 2021-04-11 22-55-44
Screenshot from 2021-04-11 22-55-29

### Demo
https://user-images.githubusercontent.com/20423590/114333958-9158c880-9b17-11eb-9a17-b77290cbbd05.mov

### CODE REVIEW

File: client/src/pages/SendResetPasswordEmailModal.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    <>
+      <LogoModal
+        title={"Reset Your Password"}
+        subtitle={"Enter your email to continue"}
+        show={!resetPasswordEmailSent}
+        handleClose={handleClose}
+        body={
+          <div className="send-reset-password-email-modal">
+            <p className="send-reset-password-email-modal-instructions">
+              If your email is registered we’ll  send a password reset link to the email address below where you can set a new password.
```
Comment (chamod-gamage) [inline client/src/pages/SendResetPasswordEmailModal.tsx]: Nit: extra space - "we'll" + "  " + " " + "send"

File: client/src/pages/SendResetPasswordEmailModal.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          <div className="text">
+            Check your email and click on the password reset link to reset the password to your account.
+            < div className="text" >
+              The link will expire after X days, after which you’ll have to request another reset link.
+            </div >
+          </div >
+          <button className="button" onClick={handleClose}>
+            I understand
+        </button>
+        </span >
```
Comment (chamod-gamage) [inline client/src/pages/SendResetPasswordEmailModal.tsx]: Nit: Incosistent whitespace inside html tags

File: client/src/services/auth.ts [hunk 3]
```diff
@@ -64,6 +66,28 @@ export const createNewAccount = async (
...
+    .sendPasswordResetEmail(email)
+    .then(() => "")
+    .catch((err) => {
+      const code: keyof AuthErrorMessageInterface = err.code;
+      return AuthErrorMessage[code];
+
+    });
+
+  return error;
+}
```
Comment (chamod-gamage) [inline client/src/services/auth.ts]: Nit: Unnecessary whitespace at the end here

**Review by chamod-gamage — APPROVED** [review]
Few nitpicks but otherwise lgtm!

File: client/src/pages/SendResetPasswordEmailModal.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+              </button>
+            </form>
+          </div >
+        } />
+      <LogoModal show={resetPasswordEmailSent} title={"Email Sent"} subtitle={''} handleClose={handleClose} body={
+        <span>
+          <div className="text">
+            Check your email and click on the password reset link to reset the password to your account.
+            < div className="text" >
+              The link will expire after X days, after which you’ll have to request another reset link.
```
Comment (azissac) [inline client/src/pages/SendResetPasswordEmailModal.tsx:79]: Change X days

**Review by azissac — CHANGES_REQUESTED** [review]
Shouldn't the X days be changed after email is sent

**Comment by bonnie-chin** [comment]
Design ✅ approved 😄

**Review by bonnie-chin — APPROVED** [review]

**Review by azissac — APPROVED** [review]
Lgtm :))

**Review by Anand-Issac — APPROVED** [review]
lgtm :))

**Pull request merged**